examples/basic_cuda_example.py

```diff
@@ -0,0 +1,179 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+"""
+Accelerated video decoding with NVDEC
+=====================================
+
+.. _nvdec_tutorial:
+
+**Author**: `Ahmad Sharif <ahmads@meta.com>`__
```

Comment:
I don't recommend adding authorship, we don't do it in the other tutorials. Over time the docs are written and re-written by many contributors, so this usually isn't accurate.

Reply:
Good point. I copied this from audio and kept it. I deleted it now